docs/clickhouse-ingress.md

```diff
@@ -0,0 +1,166 @@
+need to expose it using an Ingress resource. Create an Ingress manifest file
+(e.g., clickhouse-ingress.yaml) with the following content:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: clickhouse-ingress
+  namespace: flow-visibility   # Replace with your desired namespace
+spec:
+  ingressClassName: nginx   # Replace with your desired namespace or omit it for using default ingress class
+  rules:
+    - host: clickhouse.example.com   # Replace with your desired domain/subdomain
+      http:
+        paths:
+          - path: /
+            pathType: Prefix
+            backend:
+              service:
+                name: clickhouse-clickhouse    # Replace with your ClickHouse service name
+                port:
+                  number: 8123             # Replace with the ClickHouse service port
+  tls: # optional
+    - hosts:
+        - clickhouse.example.com       # Replace with your desired domain/subdomain
+      secretName: clickhouse-tls-secret   # Replace with the name of your TLS secret
+```
+
+Save the file and deploy the Ingress resource using kubectl:
+
+```bash
+kubectl apply -f clickhouse-ingress.yaml
+```
+
+Ensure that the Ingress resource is created successfully:
+
+```bash
+kubectl get ingress clickhouse-ingress
+```
+
+## Step 3: Create TLS Secret (Optional)
+
+If you don't have a TLS secret already, you need to create one. A TLS secret
+contains the SSL/TLS certificate and private key used for secure HTTPS
+connections. You can obtain a TLS certificate from a trusted certificate
+authority or generate a self-signed certificate for testing purposes.
+
+To create a TLS secret, use the following command:
+
+```bash
+kubectl create secret tls clickhouse-tls-secret --cert=path/to/cert.crt --key=path/to/key.key
+```
+
+Replace clickhouse-tls-secret with the desired name for the secret, and
+path/to/cert.crt and path/to/key.key with the paths to your TLS certificate
+and private key files.
+
+## Step 4: Configure Antrea Flow Aggregator
+
+To access ClickHouse using the hostname defined in the Ingress
+(e.g., clickhouse.example.com), you must configure `databaseURL` and add
+`hostAliases` to your Flow Aggregator yaml file.
+
+Change `databseURL` in `flow-aggregator.conf`:
+
+```yaml
+flow-aggregator.conf: |  
+  # clickHouse contains ClickHouse related configuration options.
+  clickHouse:
+    # Enable is the switch to enable exporting flow records to ClickHouse.
+    enable: true
+
+    # Database is the name of database where Antrea "flows" table is created.
+    database: "default"
+
+    # DatabaseURL is the url to the database. Provide the database URL as a string with format
+    # <Protocol>://<ClickHouse server FQDN or IP>:<ClickHouse port>. The protocol has to be
+    # one of the following: "tcp", "tls", "http", "https". When "tls" or "https" is used, tls
+    # will be enabled.
+    databaseURL: "http://clickhouse.example.com"
+```
+
+Add `hostAliases` in `flow-aggregator` Deployment:
```

Comment:
Shall we also expose this hostAliases option in values file for flow aggregator instead of asking user to directly modify the yaml file?

Reply:
I'm not sure if it is a common use case? I think they will still need to regenerate the yaml file in order to apply the changes?

Reply:
Just feel it might be inconvenient for the helm chart user as they have to download the source code to add this hostAliases field. If we provide this option in values file they can install FA with this enabled through official helm chart and customized value file.

Reply:
Yes, I feel the same way. I've opened the PR for hostAliases option : https://github.com/antrea-io/antrea/pull/5386.